### PR TITLE
Add link on website to new PyNEST templates and update citations

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR reorganizes the community and contribution pages to provide a link to the PyNEST template merged in #1139 so they are discoverable on ReadtheDocs website.
In addition the Zenondo citations on rtd were updated.

### CODE REVIEW

File: doc/citing-nest.rst [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
-
-For all versions below NEST v2.8.0 and for citing NEST without referring
-to a specific version, please use: `Gewaltig M-O & Diesmann M (2007)
-NEST (Neural Simulation Tool) Scholarpedia
+
+For the full citation and additional formats follow the doi Zenodo link.
+
+Cite the version you used in your work:
+
+Jordan, Jakob et al. (2019). **NEST 2.18.0** Zenodo. http://doi.org/10.5281/zenodo.2605422
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Jordan, Jakob et al. (2019). **NEST 2.18.0** Zenodo. https://doi.org/10.5281/zenodo.2605422
```

File: doc/citing-nest.rst [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
+
+Linssen, Charl et al. (2018). **NEST 2.16.0** Zenodo. http://doi.org/10.5281/zenodo.1400175
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Linssen, Charl et al. (2018). **NEST 2.16.0** Zenodo. https://doi.org/10.5281/zenodo.1400175
```

File: doc/citing-nest.rst [hunk 3]
```diff
@@ -1,34 +1,28 @@
...
+
+Peyser, Alexander et al. (2017). **NEST 2.14.0** Zenodo. http://dx.doi.org/10.5281/zenodo.882971
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Peyser, Alexander et al. (2017). **NEST 2.14.0** Zenodo. https://dx.doi.org/10.5281/zenodo.882971
```

File: doc/citing-nest.rst [hunk 4]
```diff
@@ -1,34 +1,28 @@
...
+
+Kunkel, Susanne et al. (2017). **NEST 2.12.0** Zenodo. http://dx.doi.org/10.5281/zenodo.259534
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Kunkel, Susanne et al. (2017). **NEST 2.12.0** Zenodo. https://dx.doi.org/10.5281/zenodo.259534
```

File: doc/citing-nest.rst [hunk 5]
```diff
@@ -1,34 +1,28 @@
...
+
+Bos, Hannah et al. (2015). **NEST 2.10.0** Zenodo. http://dx.doi.org/10.5281/zenodo.44222
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Bos, Hannah et al. (2015). **NEST 2.10.0** Zenodo. https://dx.doi.org/10.5281/zenodo.44222
```

File: doc/citing-nest.rst [hunk 6]
```diff
@@ -1,34 +1,28 @@
...
+
+Eppler, Jochen Martin et al. (2015). **NEST 2.8.0** Zenodo. http://dx.doi.org/10.5281/zenodo.32969
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
Eppler, Jochen Martin et al. (2015). **NEST 2.8.0** Zenodo. https://dx.doi.org/10.5281/zenodo.32969
```

File: doc/citing-nest.rst [hunk 7]
```diff
@@ -1,34 +1,28 @@
-Citing NEST
------------
+Cite NEST
+==========
```
Comment (jougs) [inline doc/citing-nest.rst]:
```suggestion
=========
```

File: doc/community.rst [hunk 8]
```diff
@@ -1,59 +1,44 @@
...
-
-Please see the `NEST developer
-space <http://nest.github.io/nest-simulator>`_ for information about
-the development workflow of NEST and for how to create a fork of our Git
-repository and make pull requests against it.
-
-Reporting bugs
---------------
+Every two weeks, we have an open video meeting to discuss current issues and developments in NEST.
+We welcome our users to join who may have questions regarding their implementations or issues they want help solving.
```
Comment (jougs) [inline doc/community.rst]:
```suggestion
We welcome users with questions regarding their implementations or issues they want help solving to join.
```

File: doc/community.rst [hunk 9]
```diff
@@ -1,59 +1,44 @@
...
+Publications using NEST
+-------------------------
 
-1. Release version or Git revision of NEST.
+We have compiled  a `list of NEST-related peer-reviewed publications <https://www.nest-simulator.org/publications/>`_ that
+we update regularly.
 
-2. Platform and operating system version.
+If you have used NEST in your research, let us know!
+Don't forget to :doc:`cite NEST <citing-nest>` in your work.
```
Comment (jougs) [inline doc/community.rst]: After the change of the Headline in the first line of this PR, should this not rather be the following?
```suggestion
Don't forget to :doc:`cite NEST <cite-nest>` in your work.
```

Comment (jessica-mitchell) [reply doc/community.rst]: citing-nest is the name of the file and not the header name

**Pull request merged**